src/store/sqlite.rs

```diff
@@ -0,0 +1,245 @@
+// This file is part of Astarte.
+//
+// Copyright 2023 SECO Mind Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//! Provides functionality for instantiating an Astarte sqlite database.
+
+use std::str::FromStr;
+
+use async_trait::async_trait;
+use log::{debug, error, trace};
+use sqlx::sqlite::{SqliteConnectOptions, SqlitePoolOptions};
+
+use super::{PropertyStore, StoredProp};
+use crate::{
+    payload::{self, Payload, PayloadError},
+    types::AstarteType,
+};
+
+/// Error returned by the [`SqliteStore`]
+#[non_exhaustive]
+#[derive(Debug, thiserror::Error)]
+pub enum SqliteError {
+    /// Error returned when the database uri is not valid
```

Comment:
Missing "." at the end of the doc comments

Reply:
Done